src/serialize-json.js

```diff
@@ -11,7 +11,7 @@
  */
 
 /* eslint-disable no-use-before-define */
-
+const { URL } = require('url');
```

Comment:
@koraa using the `URL` class from the `url` module is backward compatible:

```
$ node -v
v10.15.3
$ node -pe 'require("url").URL === URL'
true
```

Reply:
Awesome! Didn't know that even existed :)